docker-compose.yml

```diff
@@ -11,11 +11,18 @@ services:
       - 80
     volumes:
       - .:/application/code:rw
+  pgadmin:
+    image: thajeztah/pgadmin4
+    ports:
+      - "5050:5050"
+    links:
+      - database
   database:
     image: postgres:10.1
     environment:
+      POSTGRES_USER: postgres
       POSTGRES_PASSWORD: secret_password
     ports:
-      - 5432
+      - "5432:5432"
```

Comment:
Same here, using `5432` and letting a random local port be allocated to avoid conflicts with other apps also using `postgres`.